sdk/keyvault/keyvault-certificates/samples-dev/helloWorld.ts

```diff
@@ -7,9 +7,11 @@
 
 import { CertificateClient, DefaultCertificatePolicy } from "@azure/keyvault-certificates";
 import { DefaultAzureCredential } from "@azure/identity";
+import { UpdateCertificatePropertiesOptions, CertificatePolicy } from "../src/certificatesModels";
```

Comment:
```suggestion
import { DefaultAzureCredential, UpdateCertificatePropertiesOptions, CertificatePolicy } from "@azure/identity";
```

We definitely don’t want to link files from our source in our samples.